README.md

```diff
@@ -33,9 +33,20 @@ To install PyTorch/XLA nightly build in a new TPU VM:
 
 ```
 pip3 install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
+
```

Comment:
there is already a section called `Use nightly build after 08/20/2024`

Reply:
thanks, found it, sure, let's close this PR